cmd/posix.go

```diff
@@ -1079,16 +1079,13 @@ func (s *posix) CreateFile(volume, path string, fileSize int64, r io.Reader) (er
 	defer s.pool.Put(bufp)
 
 	n, err := io.CopyBuffer(w, r, *bufp)
```

Comment:
it should return the error here else it will return errLessData instead of the actual error

Reply:
Ah yes.